cmake/toolchain/arcmwdt/Kconfig.defconfig

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Synopsys
+# SPDX-License-Identifier: Apache-2.0
+
+config PICOLIBC_SUPPORTED
+	bool
```

Comment:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#common-kconfig-shorthands
> For a symbol defined in multiple locations (e.g., in a Kconfig.defconfig file in Zephyr), it is best to only give the symbol type for the “base” definition of the symbol, and to use default (instead of def_<type> value) for the remaining definitions.